app/src/main/java/org/simple/clinic/newentry/PatientEntryUpdate.kt

```diff
@@ -40,6 +40,7 @@ class PatientEntryUpdate(
       is SaveClicked -> onSaveClicked(model)
       is PatientEntrySaved -> next(model.buttonStateChanged(ButtonState.SAVED), OpenMedicalHistoryEntryScreen)
       is InputFieldsLoaded -> dispatch(SetupUi(event.inputFields))
+      is ColonyOrVillagesFetched -> next(model.colonyOrVillageListUpdated(event.colonyOrVillages))
```

Comment:
We should ideally be having 2 different commits for handling the effect and update function.